src/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -0,0 +1,198 @@
+    <value>Expected either a '\n' or '*/' to indicate end of comment, but instead reached end of data.</value>
+  </data>
+  <data name="EndOfStringNotFound" xml:space="preserve">
+    <value>Expected a '"' to indicate end of string, but instead reached end of data.</value>
+  </data>
+  <data name="ExpectedDigitNotFound" xml:space="preserve">
+    <value>Invalid number. Last character read: '{0}'. Expected a digit ('0'-'9').</value>
+  </data>
+  <data name="ExpectedDigitNotFoundEndOfData" xml:space="preserve">
+    <value>Invalid number. Last character read: '{0}'. Expected a digit ('0'-'9') but reached end of data.</value>
+  </data>
+  <data name="ExpectedEndAfterSingleJson" xml:space="preserve">
+    <value>Expected end of json after a single value but additional data found. Last character read: '{0}'.</value>
+  </data>
+  <data name="ExpectedEndOfDigitNotFound" xml:space="preserve">
+    <value>Invalid end of number. Last character read: '{0}'. Expected a delimiter.</value>
+  </data>
+  <data name="ExpectedFalse" xml:space="preserve">
+    <value>Expected a 'false' value, but instead got '{0}'.</value>
+  </data>
+  <data name="ExpectedNextDigitComponentNotFound" xml:space="preserve">
+    <value>Invalid number. Last character read: '{0}'. Expected '.' or 'E' or 'e'.</value>
+  </data>
+  <data name="ExpectedNextDigitEValueNotFound" xml:space="preserve">
+    <value>Invalid number. Last character read: '{0}'. Expected 'E' or 'e'.</value>
+  </data>
+  <data name="ExpectedNull" xml:space="preserve">
+    <value>Expected a 'null' value, but instead got '{0}'.</value>
+  </data>
+  <data name="ExpectedSeparaterAfterPropertyNameNotFound" xml:space="preserve">
+    <value>Expected a ':' following the property name, but instead got '{0}'.</value>
+  </data>
+  <data name="ExpectedStartOfPropertyNotFound" xml:space="preserve">
+    <value>Expected '"' for start of property name. Instead reached '{0}'.</value>
+  </data>
+  <data name="ExpectedStartOfPropertyOrValueNotFound" xml:space="preserve">
+    <value>Expected a start of a property name or value after ',', but reached end of data instead.</value>
+  </data>
+  <data name="ExpectedStartOfValueNotFound" xml:space="preserve">
+    <value>Expected start of a value, but instead got '{0}'.</value>
+  </data>
+  <data name="ExpectedTrue" xml:space="preserve">
+    <value>Expected a 'true' value, but instead got '{0}'.</value>
+  </data>
+  <data name="ExpectedValueAfterPropertyNameNotFound" xml:space="preserve">
+    <value>Expected a value following the property, but instead reached end of data.</value>
+  </data>
+  <data name="FoundInvalidCharacter" xml:space="preserve">
+    <value>Expected either ',', '}}', or ']', but instead got '{0}'.</value>
+  </data>
+  <data name="InvalidCharacterWithinString" xml:space="preserve">
+    <value>Found an invalid character within a json string: '{0}'. It is expected that the string is properly escaped.</value>
```

Comment:
`It is expected` is passive voice. Suggest eg `The string should be correctly escaped.` or something.